.github/actions/bpftrace/scripts/check-ipsec-leaks.bt

```diff
@@ -165,6 +165,7 @@ kprobe:br_forward
 }
 
```

Comment:
> This might cause problems in egress-gateway tests, since pod-to-outside and pod-to-node traffic is generated.

Pod-to-outside and pod-to-node traffic is also generated without egress gateway enabled. Did you mean something more specific, like this traffic is generated on the overlay when egress gateway is enabled?

Reply:
(`$trace_override` can have a better name `$from_proxy`, or more specific after this PR: `$pod_to_pod_via_proxy`)

Check-ipsec-leak.bt script is reporting errors for plain-text packets captured from bridge, as long as:
1. the packets have pod IP as both src and dst
2. or the packets have either src or dst IP inside pod CIDR, plus the packets are from proxy ($trace_override)

Let's focus on 2.

In normal case of pod->world via L7, proxy emits packets like `pod->world`, but masquerading (nf / bpf) snats the packets to `node->world`, therefore check-ipsec-leak.bt never saw pod IP on bridge, no errors were reported.

In the case of egress gateway via L7, I believe the packets must skip the masquerading, then check-ipsec-leak.bt see packets with src pod IP && from_proxy, triggers the (false) alarm. 

This PR is to make sure those traffic won't be marked as `from_proxy` ($trace_override).